### PR TITLE
STEM Standalone - Alert boxes not being read on certain browsers and screenreaders

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/11970#issuecomment-700293418
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/10203/containers/ConfirmEligibilityView.jsx [hunk 1]
```diff
@@ -72,7 +72,8 @@ export class ConfirmEligibilityView extends React.Component {
   };
 
   renderChecks = () => (
-    <div role="alert">
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```
[the remark this answers is not in the document]
Comment (zurbergram) [reply src/applications/edu-benefits/10203/containers/ConfirmEligibilityView.jsx:75]: https://github.com/department-of-veterans-affairs/va.gov-team/issues/11970#issuecomment-721906927

File: src/applications/edu-benefits/10203/containers/InitialConfirmEligibilityView.jsx [hunk 2]
```diff
@@ -11,9 +11,10 @@ function InitialConfirmEligibilityView(props) {
   captureEvents.ineligibilityAlert(props);
 
   return (
-    <div>
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```
[the remark this answers is not in the document]
Comment (zurbergram) [reply src/applications/edu-benefits/10203/containers/InitialConfirmEligibilityView.jsx:14]: https://github.com/department-of-veterans-affairs/va.gov-team/issues/11970#issuecomment-721906927

**Review by 1Copenut — APPROVED** [review]
LGTM. I'll retest in Staging when you're ready.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**